.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -56,6 +56,7 @@ Base links to copy
 
 ## Approval for review
 
+- [ ] DEV: Changes are reflected in the generated API file of the frontend and have been tested with a separate PR.
```

Comment:
would suggest to be more specific
- [ ] DEV: if api was changed - `generate-client:server` was executed in vue frontend and changes were tested and put in a PR with the same name

Reply:
that sounds better, is it ok to put the specific name of the command here in case it changes later and maybe someone forgets to change it here?

Reply:
Sorry for not replying earlier...  good point.